examples/tutorials/research/01_introduction_to_research.ipynb

```diff
@@ -0,0 +1,2472 @@
+{
```

Comment:
<p>Specify name here so a reader can easily find the folder in the output below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>